### ISSUE TITLE
Dotnet@7 not available

### ISSUE THREAD

**Issue by sladg** [timkurvers/valheim-macos#74]
Hey! 
It seems like `dotnet@7` is no longer available on Brew. Can't find anywhere as older version.

Workaround:
```
brew tap isen-ng/dotnet-sdk-versions
brew install --cask dotnet-sdk7-0-400
```
See: https://github.com/isen-ng/homebrew-dotnet-sdk-versions

**Comment by ellkrauze** [comment]
I had the same problem when installing using `brew`. My workaround was to use the .NET Runtime installer from their official website. This method is also suggested in README (see https://github.com/timkurvers/valheim-macos?tab=readme-ov-file#prerequisites)

Download and run: https://dotnet.microsoft.com/en-us/download/dotnet/thank-you/runtime-7.0.14-macos-arm64-installer

**Comment by mattcocca** [comment]
I just tried it with just `brew install dotnet` which gave me dotnet 8.0.100 and it seemed to work fine on that version as well.

**Comment by timkurvers** [comment]
Thanks for reporting! 🙏  Hope to get rid of the `dotnet` dependency in the near future.

**Comment by timkurvers** [comment]
Proposed change in #76 drops the `dotnet` requirement, hope to use that as the main method of downloading Valheim in the future.

**Comment by timkurvers** [comment]
Addressed via #76 by dropping `dotnet` requirement.
